sh/build-init-database.sh

```diff
@@ -36,4 +36,6 @@ echo "[ build ] default data"
 mysql -u $DB_USER -p$DB_PASS $DB_NAME < server/models/icd10.sql &> /dev/null
 mysql -u $DB_USER -p$DB_PASS $DB_NAME < server/models/bhima.sql &> /dev/null
 
+echo "[build] role management procedures"
+mysql -u $DB_USER -p$DB_PASS $DB_NAME <  server/models/procedures/roles.sql > /dev/null
```

Comment:
All of our procedures are built from [procedures.sql](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/models/procedures.sql).  Could you add `source` there instead of building from the .sh file?